cli/src/commands/mod.rs

```diff
@@ -140,6 +140,13 @@ enum Commands {
     Workspace(workspace::WorkspaceCommands),
 }
 
+/// A dummy command that accepts any arguments
+#[derive(clap::Args, Clone, Debug)]
+struct DummyCommand {
```

Comment:
nit: I'd call it `DummyArgs`.

Reply:
Thanks, good point! I think I slightly prefer `DummyCommandArgs`, for the least possible ambiguity.